core/src/main/java/de/evoila/cf/broker/controller/utils/DashboardAuthenticationRedirectBuilder.java

```diff
@@ -25,8 +25,9 @@
 	private String scopes;
 	
 	public DashboardAuthenticationRedirectBuilder(Dashboard dashboard, DashboardClient dashboadClient, String redirectUri,
-			String scopes) {
-		super();
+			String scopes) throws IllegalArgumentException {
+		if (dashboard == null || dashboadClient == null)
```

Comment:
I personally prefer braces for single statement if blocks, as it's more failsafe in the long run.